flink-connector/flink-connector-gcp-pubsub/src/main/java/com/google/pubsub/flink/PubSubSink.java

```diff
@@ -75,6 +85,10 @@ public abstract Builder<T> setSerializationSchema(
 
     public abstract Builder<T> setCredentials(Credentials credentials);
 
+    public abstract Builder<T> setEnableMessageOrdering(Boolean enableMessageOrdering);
```

Comment:
if a caller sets these values, they'll only be applied when a new publisher is created, correct? if so, we should document that caveat because the the name of the setter kinda implies an update

Reply:
same with the subscriber

Reply:
This method is in PubSubSink.Builder, so it can only be called before any publishers are created and cannot be used to update Pub/Sub sink behavior after it has started (at least from my understanding of Flink).

This is the same naming convention used by the underlying CPS client library, which uses a builder pattern to configure immutable properties for publisher/subscriber clients. Do you have any naming suggestions to clarify the intent?

I agree with documenting behavior/expectations.

Reply:
yeah that's my understanding as well. i think leaving the naming as is is fine and just adding a comment describing the behavior

Reply:
Done.